tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -378,7 +384,6 @@ type WorkflowContext =
 
 let SaveAndCheckProject project checker =
     async {
-        validateFileIdsAreUnique project
```

Comment:
This was there to ensure "fileId" is a unique identifier. Might lead to some issues if we relax that...

Reply:
Well this is in order to test the claim that IDE features do not start falling apart when this happens...
If real users can add the same file twice, the tests should be able to simulate it too :)